content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -16,8 +16,8 @@ v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ### Features
 
 - Use the latest version of InfluxQL package.
-- Add `-lponly` flag to [`influx export`](/influxdb/v2.0/reference/cli/influx/export/).
-- Add the ability to track number of values written via the [/debug/vars HTTP endpoint](/influxdb/v1.8/tools/api/#debug-vars-http-endpoint).
+- Add `-lponly` flag to [`influx_inspect export`](/influxdb/v1.8/tools/influx_inspect/#export).
+- Add the ability to track number of values (fields) written via the [/debug/vars HTTP endpoint](/influxdb/v1.8/tools/api/#debug-vars-http-endpoint).
```

Comment:
I would remove `(fields)` here.... and add a link to the description and name of the metric.

[track number of values](https://docs.influxdata.com/platform/monitoring/influxdata-platform/tools/measurements-internal/#valueswrittenok)